### DISCUSSION TITLE
Version 11.2

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#17430] [General]
Visual Studio tells me that there is an Avalonia Nuget update: 11.2
Is there a change log, or a thread here, describing the changes and maybe the breaking changes or known issues if any? 

Thanks!

**Comment by stevemonaco** [comment]
There's a change log with commits: https://github.com/AvaloniaUI/Avalonia/releases/tag/11.2.0

There's no blog post, AFAIK. Not sure if there will be one as new 11.x minor versions should be roughly quarterly to avoid the problem of 11.1 taking roughly a year.
